cmake/onnxruntime_webassembly.cmake

```diff
@@ -102,8 +102,11 @@ if (onnxruntime_BUILD_WEBASSEMBLY_STATIC_LIB)
       onnxruntime_flatbuffers
       onnxruntime_framework
       onnxruntime_graph
+      ${ONNXRUNTIME_XNNPACK_SCHEMAS_LIBRARY}
```

Comment:
This block is to build ONNX Runtime WebAssembly static library, i.e., libonnxruntime_webassembly.a. There is another block to build WebAssembly, i.e., ort-wasm.wasm, for web scenarios below. These libraries need to be added to there, too.